### ISSUE TITLE
Is this library compatible on android?

### ISSUE THREAD

**Issue by katastreet** [ethereum/ethereumj#1129]

**Comment by zilm13** [comment]
@katastreet we are not testing it to be compatible with android so I guess you will need to make some adaption

**Comment by mkalinin** [comment]
@katastreet could you please share some details of your use case?

**Comment by katastreet** [comment]
i was using the crypto package of this library to generate address for android.

**Comment by mkalinin** [comment]
So, you need SC security provider and crypto package. Would you like to get it as a separate library placed in some artifactory?

**Comment by katastreet** [comment]
yes exactly can i get it with changed compatibility to SC somewhere? Tried web3j but was unable to generate address direct from public key only.

**Comment by mkalinin** [comment]
EtherumJ's crypto is already based on SC rather than BC. I have created an issue reflecting your use case https://github.com/ethereum/ethereumj/issues/1130. We're planning to make some steps toward modularity when there will be capacity for that.

**Comment by katastreet** [comment]
looks like the work is done. Can i know on which branch or where?

**Comment by mkalinin** [comment]
Unfortunately no. This is the issue addressing the update we have discussed https://github.com/ethereum/ethereumj/issues/1130. I'll also reopen this one for your convenience
